src/ape_init/_cli.py

```diff
@@ -33,6 +33,29 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        git_ignore_path = project_folder / ".gitignore"
+        if git_ignore_path.exists():
+            cli_ctx.logger.warning(f"'{git_ignore_path}' exists")
+        else:
+            body = """
+# User Specified Ignored
+
+
+# Ape stuff
+.build/
+.pytest_cache
+__pycache__
+.env
+.cache
+
+# Environments
+.env
+.venv
+"""
+            with git_ignore_path.open("w", encoding="utf-8") as f:
+                f.write(body)
+                f.close()
```

Comment:
If using the `with` statement, you wouldn't need to call `close()`, but also see my other comment